components/automate-ui/src/app/pages/notifications/notifications.component.html

```diff
@@ -25,28 +25,52 @@
               <chef-tr>
                 <chef-th class="sort" tabindex="0"
                   role="button"
-                  (keyup.enter)="toggleSort('node_name')"
-                  (click)="toggleSort('node_name')">
-                  <div class="text-and-sort-icons">
-                    <span>Notification Name</span>
-                    <chef-icon aria-hidden="true" id="{{sortIcon('node_name')}}" class="sort-icon">arrow_drop_up</chef-icon>
-                  </div>
+                  (keyup.enter)="toggleSort('name')"
+                  (click)="toggleSort('name')">
+                  <span>Name</span>
+                  <chef-sort-toggle sort="name" [order]="sortIcon('')"></chef-sort-toggle>
```

Comment:
I've noticed this now in a few places in automate when using `chef-sort-toggle` and it's not your doing but I wanted to bring it up since its presenting itself.

Sort toggle is a little misleading because by default it _looks_ like there should be 2 states - "up" and "down", however it _actually_ has 3 states. An "up" a "down" and what I think is maybe a "default".  The problem presents itself when we treat it as a two state toggle.  Check out the attached video and keep an eye on what webhook type is on top, and where the arrows are.  You'll notice that eventually the arrows get out of sync with the actual sort style.  I don't know the fix off the top of my head - maybe its making a two state component 🤷‍♂️ 

![sort toggle](https://user-images.githubusercontent.com/16737484/82939688-8b321b80-9f48-11ea-943f-7b32507bdd8f.gif)

Reply:
`chef-sort-toggle` have default behavior like this.

Reply:
this doesn't appear to be fixed for me yet.  i'm still seeing the same action.

Reply:
I just fetched a new version of your branch but i'm still seeing the same action with the arrows going in and out of sync.  Is there a different action I should be seeing?
![arrows out of sync](https://user-images.githubusercontent.com/16737484/83281940-b6597c80-a18d-11ea-932d-3c768655fa28.gif)

Reply:
New issue opened here: https://github.com/chef/automate/issues/3837

 👍